identity/src/main/java/com/android/identity/DataTransportWifiAware.java

```diff
@@ -672,7 +673,7 @@ void readFromSocket(boolean isListener, Socket socket) {
                     return;
                 }
                 Log.d(TAG, "read line '" + line + "'");
-                if (line.toLowerCase().startsWith("content-length:")) {
+                if (line.toLowerCase(Locale.US).startsWith("content-length:")) {
```

Comment:
This is a specification-defined text compare (looks like HTTP to me). ASCII only please :)